docs/guide-to-triaging.md

```diff
@@ -0,0 +1,158 @@
+- **Be sceptical.** Users can feel very strongly about these issues. You will quite often see claims such as "it has always been this way" or "I've played my class since vanilla and this is how it should be." These assertions, with no other evidence, should be treated with an appropriate level of scepticism. An experienced player will probably know things about their class that you don't, but that does not mean their statements should be accepted as fact without any other evidence.
+- **Combine similar problems** - if you find related problems while searching, consider linking the two with a note. (It can be as simple as "See also <ticket URL>." This will help whoever ends up fixing the issue by telling them of similar issues.) 
+- **Try to prioritize** - more severe bugs over lesser ones, frequent bugs over rare ones. Counter-intuitively, newer issues may need priority over older ones, on the grounds that if a bug has lasted a long time without being fixed, there's a goood chance it's not very important.
+- If you can't find any sources, at least say so instead of saying nothing.
+- **Be polite.** When someone submits a bug report, they are doing us a favour. They've taken the time to make a GitHub account and type up a bug report to try to help us. The least we can do is be professional and courteous in return. 
+
+## Sources
+This is a general (and by no means exhaustive) look at the sources we can use to try to understand if a bug is valid or not. They include:
+
+- [TBC Wowhead](http://tbc.wowhead.com/) or [Current Wowhead](https://www.wowhead.com/) - Wowhead is a crucial source of data, both because of the site's own data and the comments that users have made on it. The Wowhead comments are probably the largest single set of information on individual quests and items we have, which is why it's a pity that so much of it is utter rubbish. 
+    However, it is a bit too easy to fall into the assumption that Wowhead data is correct and complete when it really isn't. Wowhead's data comes from user submissions, and has its own issues with things like drop rates as a result. For all its apparent authority, WH data can be fragmentary, incomplete and misleading if not interpreted with care.
+	Note the two versions listed here. TBC Wowhead is usually more helpful, but the current live version of Wowhead does also contain user comments from the Wrath era, which TBC WH doesn't.  
+
+- [Wowpedia](https://wowpedia.fandom.com/wiki/Special:Search) - Wowpedia can be a pretty good source, as it has a history function that lets you view an article as it was during the WotLK era. To access it, go to the relevant article, and look to the right of the article title. Next to 'View Source' you will see 3 vertically arranged dots. Clicking that allows access to the History of the article. You typically want the last article written before the date of 12 October 2010 (Cata's release date).
```

Comment:
good. Add WoWWiki too.

Reply:
I thought WowWiki was basically dead and taken over? It doesn't seem to have any content that isn't on Wowpedia.

Reply:
It is archived yes. But some pages do have more/less information than wowpedia sometimes

Reply:
Added a line with a link to WowWiki.